D2L.Services.Core.WebApi/Service.cs

```diff
@@ -26,7 +27,8 @@ public sealed class Service : IService {
 			IConfigViewer configViewer,
 			ILogProvider logProvider,
 			Action<HttpConfiguration> startup,
-			Type dependencyLoaderType
+			Type dependencyLoaderType,
+			TestDependencyRegistry testDependencyRegistry = null

```

Comment:
I'm not a big fan of optional arguments.
How about two constructors: one that takes a type, and one that takes an instance?